tests/run/genexpr_iterable_lookup_T600.pyx

```diff
@@ -35,6 +35,11 @@ def genexpr_iterable_in_closure():
     result = list( x*2 for x in x if x != 'b' )
     assert x == 'abc' # don't leak in Py3 code
     assert f() == 'abc' # don't leak in Py3 code
+
+    # Py2 cleanup (pretty irrelevant to the actual test!)
+    import sys
+    if sys.version_info[0] == 2:
+        result = map(bytes, result)
```

Comment:
I find `str()` less confusing than `bytes()` here.
```suggestion
    # Py2 cleanup (pretty irrelevant to the actual test!)
    import sys
    if sys.version_info[0] == 2:
        result = map(str, result)
```

Reply:
It looks like `bytes` is actually necessary here so I'll change this one back